components/filter/filter.js

```diff
@@ -150,6 +163,10 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 			const builtDimension = this._buildDimension(dimension);
 			return html`<d2l-menu-item text="${dimension.text}">
 				${builtDimension}
+				<div slot="supporting">
+					<span>${this._formatFilterCount(dimension.appliedCount, dimension.maxCount)}</span>
+					<d2l-offscreen>${this.localize('components.filter.filterCountDescription', { number: dimension.appliedCount })}</d2l-offscreen>
```

Comment:
I don't feel strongly about this, but there's also the `offscreenStyles` that you can use with the `d2l-offscreen` CSS class. In general it's probably better to use fewer web components/elements if possible.

Reply:
I'm actually gonna move this to the description of the menu-item for now, so that it definitely gets read out.  NVDA is ignoring this.  Not sure how this will work once the status-indicator is in - if it also will be ignored and we'll need to keep the accessible text on the menu-item anyways, or if it will be done in a way that the screenreader sees it.